build.sbt

```
@@ -55,6 +55,7 @@ lazy val `cats-effect` = module(project) in file("modules/cats-effect")
 lazy val `cats-effect2` = module(project) in file("modules/cats-effect2")
 lazy val circe = module(project) in file("modules/circe")
 lazy val cron4s = module(project) in file("modules/cron4s")
+lazy val doobieHikari = module(project) in file("modules/doobie-hikari") dependsOn (generic)
```

Nitpick: For consistency with the other references.

```suggestion
lazy val `doobie-hikari` = module(project) in file("modules/doobie-hikari") dependsOn (generic)
```

---

oh yeah, good point, thanks